frontend/ui/data/onetime_migration.lua

```diff
@@ -886,5 +886,42 @@ if last_migration_date < 20250405 then
     G_reader_settings:delSetting("show_finished")
 end
 
+-- 20250420, Remove separate "smaller footnotes" tweak
+-- https://github.com/koreader/koreader/pull/13639
+if last_migration_date < 20250420 then
+    local global_tweaks = G_reader_settings:readSetting("style_tweaks")
+
+    if global_tweaks then
+        for __, pct in ipairs( { 100, 90, 85, 80, 75, 70, 65 } ) do
+            if global_tweaks["font-size_" .. pct] then
+                global_tweaks["inpage_footnote_text_force_size"] = true
```

Comment:
The original footnote font size tweaks are named `id = T("inpage_footnote_font-size_%1", pct)`, so you're missing `inpage_footnote_` on line 896 ?